wrappers/android/tools/camera/src/main/res/values/arrays.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string-array name="controls_emitter_entries">
+        <item>"OFF"</item>
```

Comment:
This can be done dynamically with `get_option_value_description` API.

Reply:
done